docs/website/content/docs/installation.md

```diff
@@ -10,7 +10,7 @@ toc: true
 
 Sysl is a CLI (Command Line Interface) that excecutes with the `sysl` command.
 
-    go get -v github.com/anz-bank/sysl/cmd/sysl
+    GO111MODULE=on go get github.com/anz-bank/sysl/cmd@v0.4.0
```

Comment:
This also needs a note to NOT run it from inside a Go source directory that is module enabled, otherwise it gets added to `go.mod`/`go.sum`. I just tested with a fresh module: `mkdir xxx; cd xxx; go mod init xxx; GOMODULE111=on go get ...` and it updated the go modules. From a quick look of the help, I can't see any way to tell `go get` not to do this. sigh.

Reply:
Done.